fast_api/routes/attribute.py

```diff
@@ -0,0 +1,18 @@
+from controller.attribute import manager
+from typing import List, Union
+from fast_api.routes.client_response import pack_json_result
+from fastapi import APIRouter, Query
+from submodules.model.util import sql_alchemy_to_dict
+
+router = APIRouter()
+
+
+@router.get("/all-attributes/{project_id}")
```

Comment:
Should resolve to:
`@router.get("/{project_id}/attributes_by_project_id")`

Reply:
Also take note on the middleware, as we are checking for the project id in the specified order for access.